jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml

```diff
@@ -330,6 +342,9 @@
                 export JENKINS_PUBLISHED_VERSION="ci/latest-1.2"
                 export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:Ingress\]"
                 export PROJECT="kubernetes-ingress-1-2"
+                # build-1.2 is not using the kubernetes-release-dev bucket yet
```

Comment:
There's a release-1.3 ingress job, so the 1.2 job should be moved to the appropriate section.